File: week1-cpp/Image_Annotation/Drawing_Rectangles.cpp
// Include libraries
#include <iostream>
#include "dataPath.hpp"
#include <opencv2/core.hpp>
#include <opencv2/imgproc.hpp>
#include <opencv2/highgui.hpp>
using namespace std;
using namespace cv;

int main(void){

        // Path to the image we are going to read
        // This can be an absolute or relative path
        // Here we are using a relative path
        string imageName = DATA_PATH+"images/boy.jpg";

        // Load the image
        Mat image;
        image = imread(imageName, IMREAD_COLOR);

	// Draw a rectangle (thickness is a positive integer)
	Mat imageRectangle = image.clone();
	rectangle(imageRectangle, Point(170, 50), Point(300, 200),
          Scalar(255, 0, 255), 5, LINE_8);

	imwrite("../results/rectangle.jpg",imageRectangle);

	//imshow("Draw rectangle on image",imageRectangle);
	//waitKey(0);
	return 0;
}

File: week1-python/Image_Annotation/Drawing_Rectangles.py
# Import cv2 module
import cv2
from dataPath import DATA_PATH

# Path to the image we are going to read
# This can be an absolute or relative path
# Here we are using a relative path
imageName = DATA_PATH+"images/boy.jpg"

# Load the image
image = cv2.imread(imageName, cv2.IMREAD_COLOR)

# Draw a rectangle (thickness is a positive integer)
imageRectangle = image.copy()
cv2.rectangle(imageRectangle, (170, 50), (300, 200), (255, 0, 255), thickness=5, lineType=cv2.LINE_8);

# Display the image
#cv2.imshow("Image Rectangle",imageRectangle)
#cv2.waitKey(0)
cv2.imwrite("../results/imageRectangle.png",imageRectangle)
